test/e2e/framework.go

```diff
@@ -877,7 +892,23 @@ func (data *TestData) deployFlowAggregator(ipfixCollector string) error {
 	if err != nil || rc != 0 {
 		return fmt.Errorf("error when deploying the Flow Aggregator; %s not available on the control-plane Node", flowAggYaml)
 	}
-	if err = data.mutateFlowAggregatorConfigMap(ipfixCollector); err != nil {
+	// clickhouse-ca Secret is created in the flow-visibility namespace. In order to make it accessible for Flow Aggregator,
+	// we copy it from namespace flow-visibility to namespace flow-aggregator when secureConnection is true.
+	if o.secureConnection {
+		secret, err := data.clientset.CoreV1().Secrets(flowVisibilityNamespace).Get(context.TODO(), flowAggregatorCHSecret, metav1.GetOptions{})
+		if err != nil {
+			return fmt.Errorf("unable to get Secret with name %s in Namespace %s: %v", flowAggregatorCHSecret, flowVisibilityNamespace, err)
+		}
+		secret.ObjectMeta.Namespace = flowAggregatorNamespace
+		// Set to empty string to avoid error: resourceVersion should not be set on objects to be created
+		secret.ResourceVersion = ""
```

Comment:
It's a bit hacky: just create a new secret object and use the same `Data` field

Reply:
Fixed. Thanks.